ziti/enroll/enroll.go

```diff
@@ -269,31 +269,31 @@ func enrollOTT(token *config.EnrollmentClaims, cfg *config.Config, caPool *x509.
 		return err
 	}
 
-	respBody, err := ioutil.ReadAll(resp.Body)
```

Comment:
@dovholuknf I want you to look at this section. You were in here last and updated this logic - I want to make sure this is okay with what you saw when you were in here.

Reply:
Using gabs to parse the JSON allows us to not depend on ziti-edge.

Reply:
> @dovholuknf I want you to look at this section. You were in here last and updated this logic - I want to make sure this is okay with what you saw when you were in here.

we chatted. quick synopsis... @andrewpmartinez is gonna look into the causemessage/cause.message and add back the one that makes sense/is used.

Reply:
Updated. See comment lower down.